src/align.cpp

```diff
@@ -1531,10 +1531,12 @@ chunk_t *align_trailing_comments(chunk_t *start)
    size_t          min_col  = 0;
    size_t          min_orig = 0;
    chunk_t         *pc      = start;
+   const size_t    lvl      = start->brace_level;
```

Comment:
Can `start` be NULL?

Reply:
No:
https://github.com/uncrustify/uncrustify/blob/738981d167bd7c79d80bf608968e9b8a8293f7dd/src/align.cpp#L567-L577
This is the only call to `align_trailing_comments`. Besides that the loop ensures non-NULL-ness, we've already dereferenced the chunk pointer before calling `align_trailing_comments`.

Reply:
I'm on mobile and it's a bit hard for me to see everything

Reply:
No worries, it was a fair question :smile:. Better to ask me to "show my work" than assume and let a potential bug slip in.